tests/vmi_multiqueue_test.go

```diff
@@ -151,5 +152,20 @@ var _ = Describe("[sig-compute]MultiQueue", func() {
 			}
 		})
 
+		It("should report the correct queue count when requesting multiple vCPU", func() {
```

Comment:
I think we already have tests above that create a VMI with multi CPU/s, so we can add there this check. No?
Starting a VMI is costly, if we can reduce the cost it will be nice.

Reply:
merged with other test